File: 7KYU/Length and two values./solution.cs
using System.Collections.Generic;
using System.Linq;
public class Kata {
  public static object[] Alternate(int n, object firstValue, object secondValue) {
    var xs = new List<object>();
    
    for(var i = 0; i < n; i++)
    {
      xs.Add(firstValue);
      xs.Add(secondValue);
    }
    
    return xs.Take(n).ToArray();
  }
}

File: 7KYU/Length and two values./solution.go
package kata

func Alternate(n int, firstValue string, secondValue string) []string {
	var res []string

	for i := 0; i < n; i += 2 {
		res = append(res, firstValue)
		if len(res) != n {
			res = append(res, secondValue)
		}

	}

	return res
}
